src/song_utils.py

```diff
@@ -232,3 +232,41 @@ def get_cover_art(filename: str) -> Optional[File]:
 
     # Create a new discord file from the file pointer and name
     return File(image_bytes_fp, filename=cover_filename)
+
+
+def convert_timestamp_to_seconds(time_str: str):
+    # Converts a time string into seconds. Returns 0 if format is invalid.
+    # Format is handled either in pure seconds (93, 180) or hh:mm:ss format (1:23:45).
+    if time_str is None:
+        return None
+
+    if ":" not in time_str:
```

Comment:
Nit: You can remove this check and it should still have the same behavior (less efficient by a tiny amt but totally inconsequential, I think clarity is more valuable here)

Reply:
by "this check" i mean at line 243, idk why it highlighted the wrong section